app/views/catalog/explorer.html.haml

```diff
@@ -1,6 +1,6 @@
 -# Include the center cell divs
 - nodetypes = %w(MiqTemplate OrchestrationTemplate ServiceResource ServiceTemplate ServiceTemplateCatalog)
-- if nodetypes.include?(TreeBuilder.get_model_for_prefix(@nodetype)) && !@view
+- if nodetypes.include?(TreeBuilder.get_model_for_prefix(@nodetype)) && (!@view || @record.present?)
```

Comment:
This doesn't sound right.

This is essentially the same condition as in `replace_right_cell` (except for `Service` vs `ServiceRecord` in the list of types.

So... why not solve it the same way? 

Adding something like `|| (%w(accordion_select button_create group_create servicetemplate_edit reload tree_select).include?(params[:action]) && @record.present? && type == 'st')` would be ugly, but should be a 100% match, right?

OTOH I have no idea how `@record.present?` there affects all the other explorer screens...

Reply:
Or... was it that we can always expect that `@record.present?` will be false in all the explorer GTL views? (I'm not sure that's the case, but if so, then my concern is reduced to "what happens to MiqTemplate textual summary".)

Reply:
@mzazrivec Looking at the BZ, I think the bug is actually that `@view` is set and shouldn't be.

The presence of `@view` means there is a GTL set up.
According to the BZ, the target screen should not have any GTLs.

So IMO this is wrong, and we need to clear `@view` instead.